subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/ProjectLoadingIntegrationTest.java

```diff
@@ -150,6 +151,20 @@ public void buildFailsWhenSpecifiedSettingsFileIsNotAFile() {
         result.assertHasDescription("The specified settings file '" + file + "' is not a file.");
     }
 
+    @Issue("gradle/gradle#4672")
+    @Test
+    public void buildFailsWhenSpecifiedInitScriptIsNotAFile() {
+        TestFile file = testFile("unknown");
+
+        ExecutionFailure result = inTestDirectory().usingInitScript(file).runWithFailure();
```

Comment:
I guess we should have some tests where we have more than one init script argument on the command line, too. WDYT?

Reply:
Sure, could be good to have. Does this look like what you had in mind? e28340725ead1e0f223f4327ef03121ac95a085d

Reply:
👍